### ISSUE TITLE
Handle `mergeStateStatus` being `UNKNOWN`

### ISSUE THREAD

**Issue by gabrielittner** [modularml/stack-pr#54]
I'm pretty regularly running into this error when landing a stack:
```
ERROR: Associated PR is not mergeable on GitHub!
     e42b9158 (#9899, 'gabriel/stack/1' -> 'main'): ...

Please fix the issues on GitHub.

PR info from github: {'baseRefName': 'main', 'body': '...', 'headRefName': 'gabriel/stack/1', 'mergeStateStatus': 'UNKNOWN', 'number': 9899, 'state': 'OPEN', 'title': '...`', 'url': 'https://github.com/.../pull/9899'}
```

This usually happens when another PR was merged and no one looked at the PR since then. Usually the Github UI will show some kind of processing indicator. If I just re-run `stack-pr land` it then works immediately. It would be great if `stack-pr` could detect the `UNKNOWN` state and then just internally retry after waiting a few seconds.

**Comment by rvi** [comment]
I have a similar problem with the `mergeStateStatus` is `UNSTABLE`, in my case it's impossible to land the code if some optional github checks are not passing, on the other hand I can land the code on github.

**Comment by AmaranthineCodices** [comment]
This was fixed by #49 but we haven't cut a new release yet.
